### ISSUE TITLE
Missing semantic differentiation between status and heading

### ISSUE THREAD

**Issue by swashbuck** [adaptlearning/adapt-contrib-accordion#118]
## Bug Description:
After an accordion item has been visited, "visited" text is included in the item title without semantic differentiation. This can render understanding of the item title unclear for screen reader users.

## Expected Behavior:

- Update the accordion label to include its updated state after it has been visited (ex: Lists (visited))
- Add a checkmark icon to visually indicate the updated state of the item.
